rebar.config

```diff
@@ -3,7 +3,7 @@
 {deps, [
         {reltool_util, ".*", {git, "https://github.com/okeuday/reltool_util.git", {tag, "v1.4.0"}}},
         {riak_core, ".*", {git, "git://github.com/Regulators/riak_core", {tag,"2.0.1jb3"}}},
-        {rafter, ".*", {git, "git@github.com:andrewjstone/rafter.git", {branch, "master"}}}
+        {rafter, ".*", {git, "https://github.com/andrewjstone/rafter.git", {tag, "0.2.1"}}}
```

Comment:
I've requested a tag be added to rafter in branch/master: https://github.com/andrewjstone/rafter/issues/31